Dockerfile.backend-container

```diff
@@ -2,10 +2,10 @@ FROM alpine
 RUN mkdir -p /opt/spire
 
 WORKDIR /opt/spire
+ENTRYPOINT ["/opt/spire/run_backend.sh"]
 
-COPY bin/tornjak-backend tornjak-backend
 COPY sample-keys sample-keys
-
 # Add init
 COPY run_backend.sh run_backend.sh
-ENTRYPOINT ["/opt/spire/run_backend.sh"]
+COPY bin/tornjak-backend tornjak-backend
```

Comment:
Moving this to the end makes the things most likely to change on new images in the last docker layer, meaning the layers before can all be reused as is. No pulls and pushes and rebuilds required for those layers.

Reply:
Thank you, makes sense!